R/parameters.R

```diff
@@ -19,18 +19,41 @@ get_parameters <- function(iso3c = NULL,
     contact_matrix_set <- squire::get_mixing_matrix(iso3c = iso3c)
   }
 
-  c(
-    squire::parameters_explicit_SEEIR(
+  prob_asymp <- c(0.3, 0.3, rep(0.2, 15))
+  IAsymp_0 <- c(rep(0L, 17))
+  dur_IAsymp <- 2.1
+
+  parameters <- list(
+    sq = squire::parameters_explicit_SEEIR(
       population = population,
       country = get_country(iso3c),
       contact_matrix_set = contact_matrix_set,
-      dt = 1, # dt should always be 1 as individual is always discrete time
+      dt = 1, # dt should always be 1 as individual is always a discrete time
       time_period = time_period,
       ...
     ),
+    prob_asymp = prob_asymp,
+    IAsymp_0 = IAsymp_0,
+    dur_IAsymp = dur_IAsymp,
     time_period = time_period,
     max_age = max_age
   )
+
+  # Remove contact _matrix_set from paraeter list as this is passed straight
+  # into infection_process
+  parstemp <- parameters$sq
+
+  parstemp <- parstemp[names(parstemp) != "contact_matrix_set"]
```

Comment:
This is what `remove_non_numerics` was doing in the model function. Would recommend using this function rather than just removing contact_matrix_set as this means that if we add any other non-numeric or non-string variables to the squire parameters function they will also be removed and these will cause individual to break.

Reply:
That started causing me problems and that is why I removed it. I will investigate further.